checkstyle/suppressions.xml

```diff
@@ -142,6 +142,14 @@
     <suppress checks="MethodCount" files="com.hazelcast.concurrent.lock.LockStoreImpl"/>
     <suppress checks="Todo" files="com.hazelcast.concurrent.lock.ConditionImpl"/>
 
+    <!-- ringbuffer -->
+    <suppress checks="JavadocMethod" files="com.hazelcast.ringbuffer.impl.operations[\\/]"/>
+    <suppress checks="JavadocType" files="com.hazelcast.ringbuffer.impl.operations[\\/]"/>
+    <suppress checks="JavadocMethod" files="com.hazelcast.ringbuffer.impl.operations[\\/]"/>
+    <suppress checks="JavadocVariable" files="com.hazelcast.ringbuffer.impl.operations[\\/]"/>
+    <suppress checks="JavadocPackage" files="com.hazelcast.ringbuffer.impl.operations[\\/]"/>
```

Comment:
Good idea. But I would rather do the whole checkstyle file in 1 go in a different PR.